presto-main/src/test/java/com/facebook/presto/operator/MockExchangeRequestProcessor.java

```diff
@@ -61,6 +62,8 @@
 
     private final DataSize expectedMaxSize;
 
+    List<DataSize> requestMaxSizes = Collections.synchronizedList(new ArrayList<>());
```

Comment:
private final + getter. 

We are trying not to access the non class - local fields directly.

Reply:
Done